msteams-platform/bots/what-are-bots.md

```diff
@@ -20,17 +20,17 @@ A bot is also referred to as a chatbot or conversational bot. It is an app that
 
 Conversational bots allow users to interact with your web service using text, interactive cards, and task modules.
 
-:::image type="content" source="../assets/images/invokebotwithtext.png" alt-text="Web service using text"lightbox="../assets/images/invokebotwithtext.png":::
+:::image type="content" source="../assets/images/invokebotwithtext.png" alt-text="The screenshot is an example that shows the Web service using text."lightbox="../assets/images/invokebotwithtext.png":::
 
-:::image type="content" source="../assets/images/invokebotwithcard.png" alt-text="web service using interactive cards"lightbox="../assets/images/invokebotwithcard.png"border="true":::
+:::image type="content" source="../assets/images/invokebotwithcard.png" alt-text="The screenshot is an example that shows the Web service using interactive cards."lightbox="../assets/images/invokebotwithcard.png":::
 
-:::image type="content" source="../assets/images/task-module-example.png" alt-text="web service using task module"lightbox="../assets/images/task-module-example.png"border="true":::
+:::image type="content" source="../assets/images/task-module-example.png" alt-text="The screenshot is an example that shows the Web service using task module."lightbox="../assets/images/task-module-example.png":::
 
 Conversational bots are incredibly flexible. Bots can handle a few basic commands or complex tasks that involve artificial intelligence and natural language processing. Bots can be part of a larger application or be standalone.
 
 Use the right mix of cards, text, and task modules to create a useful bot. The following image shows a user conversing with a bot in a one-to-one chat using text and interactive cards.
 
-:::image type="content" source="~/assets/images/FAQPlusEndUser.gif" alt-text="Sample FAQ bot":::
+:::image type="content" source="~/assets/images/FAQPlusEndUser.gif" alt-text="The screenshot is an example that shows the Sample FAQ bot.":::
```

Comment:
Not a screenshot. 
And it is more than that. Just read the documented text before the image.

Reply:
resolved